File: 7.Reverse_Integer/7.Reverse_Integer.py
class Solution:
    def reverse(self, x: int) -> int:
        neg = False
        if x < 0:
            neg = True
            x*= -1
        reverse = int(str(x)[::-1])
        if reverse > 0x80000000 or reverse < -0x7fffffff:
            return 0
        elif neg:
            return -reverse
        return reverse

File: 7.Reverse_Integer/7.Reverse_Integer.js
/**
 * @param {number} x
 * @return {number}
 */
var reverse = function(x) {
    var neg = false;
    if (x < 0) {
        neg = true;
        x*= -1;
    }
    var reverse = parseInt(x.toString().split("").reverse().join(""));
    if (reverse > 0x80000000 || reverse < -0x7fffffff)
        return 0;
    else if (neg)
        return -reverse;
    return reverse;
};
